other/gss/gss.c

```diff
@@ -0,0 +1,107 @@
+/*
+    Generic Sudoku Solver (gss) link to tdoku benchmark
+    Copyright (C) 2019 B. Pieters
+
+    This program is free software: you can redistribute it and/or modify
```

Comment:
Can you change the license on this file to a permissive one? Everything I'm actually including in the project is BSD or similar.